server.js

```diff
@@ -1,16 +1,17 @@
 require("dotenv").config();
 /* ==== External Modules ==== */
 const express = require("express");
-//const morgan = require('morgan');
+const morgan = require('morgan');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const morgan = require("morgan");
```